crates/findex/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 #![cfg_attr(any(feature = "wasm_bindgen", feature = "ffi"), feature(const_option))]
 #![cfg_attr(feature = "cloud", feature(iter_next_chunk))]
 
+pub mod implementations;
```

Comment:
You may hide this declaration behind the features `sqlite` or `redis`.

Reply:
ok